deps/pom.xml

```diff
@@ -38,7 +38,6 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <skipITs>true</skipITs>
```

Comment:
This "skipITs" flag has outlived its usefulness, so I'm killing it. There are maven sponsored ways to skip integration tests